components/audio_video_preview/audio_video_preview.tsx

```diff
@@ -55,7 +55,11 @@ export default class AudioVideoPreview extends React.PureComponent<Props, State>
             video = document.createElement('video');
         }
 
-        const canPlayType = video.canPlayType(fileInfo.mime_type);
+        // mov files are encoded with MPEG-4 but the mimetype is set to quicktime, should be allowed in chrome
+        // Long standing chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=1120292
+        const mimeType = (fileInfo.mime_type === 'video/quicktime') ? 'video/mp4' : fileInfo.mime_type;
+
+        const canPlayType = video.canPlayType(mimeType);
```

Comment:
So here are my thoughts 
-  both the play types `maybe` and `probably` doesnt guarantee the video will be playable 
- Instead of we checking the mime types, we can let the browser decide if its playable or not by passing the video ref directly to video element
- If the video is not playable, it throws an error which is caught `handleLoadError` which then can set `canPlay` to false
- And we have the UI for unplayable videos.
- This avoids us to spoof the mimeType above

Below is the sample code,
```js
    handleFileInfoChanged = (fileInfo: FileInfo) => {
        let video = this.videoRef.current;
        if (!video) {
            video = document.createElement('video');
        }

        this.setState({
            canPlay: true,
        });
    }
```

Reply:
Okay, I adjusted it. It also meant removing the `fileInfo` used when calling `handleFileInfoChanged`. I feel like there can be a better check for can play closer to the `render` process, but for now this should work. Let me know if you have any other thoughts.

Reply:
@Willyfrog what do you think?

Reply:
I prefer this approach 👍🏼